packages/reporter/cypress/e2e/shortcuts.cy.ts

```diff
@@ -161,5 +161,25 @@ describe('shortcuts', function () {
         expect(runner.emit).not.to.have.been.calledWith('focus:tests')
       })
     })
+
+    context('focus search', () => {
```

Comment:
Can you split this test up and/or add a log to  indicate which part is testing the spec list is open or not? It's not obvious looking through this without comparing your changes

Reply:
Can do!